Robinhood/Robinhood.py

```diff
@@ -757,3 +757,79 @@ def place_sell_order(
         """
         transaction = Transaction.SELL
         return self.place_order(instrument, quantity, bid_price, transaction)
+    
+    ##############################
+    #GET OPEN ORDER(S)
+    ##############################
+    
+    def get_open_orders(self):
+        """
+        Returns all currently open (cancellable) orders.
+        If not orders are currently open, `None` is returned.
+        TODO: Is there a way to get these from the API endpoint without stepping through order history?
+        """
+
+        open_orders = []
+        orders = self.order_history()
+        for order in orders['results']:
```

Comment:
I think this would be a little bit more succinct with a list comprehension like so:
```python
open_orders = [order for order in orders['history'] if order['cancel']]
```